app/common/enterprise-util.ts

```diff
@@ -6,6 +6,8 @@ import process from "node:process";
 import {z} from "zod";
 import {dialog} from "zulip:remote";
 
+import {ipcRenderer} from "../renderer/js/typed-ipc-renderer.js";
```

Comment:
This is worse. We should not import *either* main-only APIs or renderer-only APIs from `common`. Our linter configuration is supposed to enforce this, but that enforcement appears to be broken somehow.

Reply:
I'm not sure why this is worse. Can you explain why? Thanks :)

Reply:
#1425 incorrectly imports a main-only API into `common`. This incorrectly imports both a main-only API and also a renderer-only API into `common`. We should import neither here, not both.

I understand that you think you’re avoiding issues by checking `process.type`, but (1) a linter cannot automatically verify that you did that check correctly, (2) it’s too easy to forget to check manually, and (3) the `import` itself has side effects that can’t be conditionalized.